back-office/wishlist.md

```diff
@@ -0,0 +1,346 @@
+As a logged-in user, I can access my favorite products by:  
+- Going to my account  
+- Clicking on "Wishlists"
+![wishlist screen 6](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-6.png)
+
+- I come to the page with all my favorites lists (default one + created lists)
+![wishlist screen 7](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-7.png)
+
+- When I click on one of the lists, it brings me to a page that lists my previously added products  
+
+- The number of products in the list is displayed to the left of the list name  
+This means that it accounts for all the idproduct allocated
+For example, if there are 3 variations of the same product it counts as 3 products.
+In the case of a product pack, this counts as 1 pack and not the number of products in the pack
+
+![wishlist screen 8](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-8.png)
+
+I can view the products and I can sort them as follows:  
+- Last added (default sorting)  
+- Price highest to lowest
+- Price lowest to highest 
+
+In all three cases, keep the discount price in mind  
+![wishlist screen 9](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-9.png)
+
+Please note that if a product is deleted or deactivated from the catalog, the product automatically disappears from the list.  
+
+For each product, you have to go back to the information listed in US n°1.
+As a logged-in user, I can see how many unique favorite products are in a list.  
+
+What type of users can add products to their favorites?
+Only logged-in users, since the Favorites list is displayed in My Account.  
+
+When I access the page with the my favorites lists, I can see the number of products in each of the lists.  
+
+### US 3: Edit and/or delete a favorite product
+##### As a logged-in user, I can edit and/or delete a favorite product  
+Different cases:
+- Products in stock
+- Products out of stock
+- from the product sheet
+- from a favorites list
+
+Functional rules:
+I can modify the quantity, the combinations and the customization of all my favorite products (in stock, out of stock) by clicking on the Pencil in my favorite products list.
+
+When I click on the Pencil, I am redirected to the product sheet to make my modifications. Once the changes are made, you will have to click on the heart to add the new product as a favorite. At this point, the new quantity, combination and / or customization will overwrite the original product in the favorites list.
+
+![wishlist screen 10](https://github.com/PrestaShop/prestashop-specs/blob/master/img/wishlist-screen-10.png)
+
+I can permanently remove a favorite product from the list by clicking on remove and a pop-up appears confirming that the product has been removed from the list.  
+
+You can remove a product from the favorites directly within the product sheet or a product listing, by clicking on the heart (which becomes empty).  
```

Comment:
Already written as well, cf. 'If the user clicks on the filled-in heart again, the product is removed from the wishlist. If the product is deleted by the seller, it should no longer appear in the users' favorites lists.'.